Source/Core/InputCommon/GCAdapter.cpp

```diff
@@ -206,34 +216,47 @@ static void ReadThreadFunc()
   auto poll_rate_measurement_start_time = Clock::now();
   int poll_rate_measurement_count = 0;
 
-  while (s_read_adapter_thread_running.IsSet())
+  bool last_read_was_failing = false;
```

Comment:
```suggestion
  bool last_read_failed = false;
```

Unless there's a reason that I'm not seeing, "was_failing" seems like weird phrasing for this variable name?